dali/c_api/c_api.cc

```diff
@@ -383,13 +402,14 @@ static void daliCopyTensorNToHelper(dali::DeviceWorkspace* ws, void* dst, int n,
 }
 
 void daliCopyTensorNTo(daliPipelineHandle* pipe_handle, void* dst, int n, device_type_t dst_type,
-                       cudaStream_t stream, bool non_blocking) {
+                       cudaStream_t stream, int non_blocking) {
+  bool nb = non_blocking > 0;
```

Comment:
```suggestion
  bool nb = non_blocking != 0;
```
to be consistent with the previous ones. Also, you can also `static_cast<bool>(non_blocking)` or even pass it as an int directly